test/query.test.js

```diff
@@ -53,4 +53,66 @@ describe('Testing query route', () => {
     const res = await request(app).get(`/query?text=&filter=`);
     expect(res.status).toBe(400);
   });
+
+  test('Testing page with empty param value', async () => {
```

Comment:
Change this title to better reflect what the test is doing, namely, testing an empty `page` value.

Reply:
whoops overlooked this one, on it